### PR TITLE
Adding apirequestcount cluster read rights to sre group

### PR DESCRIPTION
**Pull request by dystewart**
@HumairAK So far I've only added the `clusterrole` to the infra and smaug cluster, I will add other clusters as needed

### CODE REVIEW

**Review by Gregory-Pereira — COMMENTED** [review]
/lgtm

File: cluster-scope/base/rbac.authorization.k8s.io/clusterroles/sre-read-cr/clusterrole.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: sre-group-read
+rules:
+- apiGroups: [""]
```
Comment (HumairAK) [inline cluster-scope/base/rbac.authorization.k8s.io/clusterroles/sre-read-cr/clusterrole.yaml:6]: can you convert these to yaml so it's consistent with the other clusterroles in this path

**Comment by HumairAK** [comment]
Can you link the issue / context of this issue? I'm not clear on what we're resolving here (I may have forgotten :smile: )

**Comment by dystewart** [comment]
This PR is in response to [https://github.com/operate-first/operations/issues/455](https://github.com/operate-first/operations/issues/455)

File: cluster-scope/base/rbac.authorization.k8s.io/clusterroles/sre-read-cr/clusterrole.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: sre-group-read
+rules:
+- apiGroups:
+  - ""
```
Comment (HumairAK) [inline cluster-scope/base/rbac.authorization.k8s.io/clusterroles/sre-read-cr/clusterrole.yaml:7]: @dystewart seems like the issue you are aiming to resolve is only asking for asking to this [resource](https://docs.openshift.com/container-platform/4.8/rest_api/metadata_apis/apirequestcount-apiserver-openshift-io-v1.html), and not all of the core group (denoted by `""`), can you scope this to just that resource? you will need to update `apiGroups` and `resources` field. You should also rename the name of this role/rolebinding to something like `apirequestcount-reader`

File: cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/apirequest-count-reader/clusterrolebinding.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (Gregory-Pereira) [inline cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/apirequest-count-reader/clusterrolebinding.yaml:1]:
```suggestion
apiVersion: rbac.authorization.k8s.io/v1beta1
apiVersion: rbac.authorization.k8s.io/v1
```

Similar to [Humair's PR](https://github.com/operate-first/apps/pull/1874) rbac.authorization.k8s.io version `v1beta1` has been depricated

Comment (dystewart) [reply cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/apirequest-count-reader/clusterrolebinding.yaml]: @Gregory-Pereira Oops missed that, updated

**Pull request merged**